### PR TITLE
456 post nav

### PR DESCRIPTION
**Pull request by RobertWDLowe**
Post Nav Fix:

Next and prev links now load through JS.

@ollietreend  and @marcincichon  please review
To confirm:

1. Are we happy that the service apis are now called get method? E.g.
http://www.moj-intranet.com/service/news/get
http://www.moj-intranet.com/service/post/get

2. Are we happy the new api endpoint and naming is siblings? E.g.
http://www.moj-intranet.com/service/news/siblings/laa//15716

3. Should we hide the next and prev links before the links are loaded in?

### CODE REVIEW

File: wp-content/themes/mojintranet/models/post_siblings.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+
+
+
+
+
+
+
+
```
Comment (marcincichon) [inline wp-content/themes/mojintranet/models/post_siblings.php]: Whitespace :-)

**Comment by marcincichon** [comment]
@RobertWDLowe Hi, I only had a limited amount of time to check the code but here's my 2 cents:
1. fine by me but we could revisit one day
2. ditto
3. Yes, they should be definitely hidden until ready. Also, can we restore the static html <a></a> instead of the dynamically generated one? Always better to keep this kind of stuff in the view.

**Comment by RobertWDLowe** [comment]
Content nav is now hidden till loaded...Also I have change it so the a tag is removed when no link is provided so that all html is in view.

File: wp-content/themes/mojintranet/modules/api/news.php [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ollietreend) [inline wp-content/themes/mojintranet/modules/api/news.php]: @RobertWDLowe Please remove whitespace from before the opening `<?php` tag.

**Comment by ollietreend** [comment]
Looks good @RobertWDLowe! 👍

**Pull request merged**